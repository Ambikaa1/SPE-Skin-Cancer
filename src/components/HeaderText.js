import React from "react";
import { Text, StyleSheet } from "react-native";

const HeaderText = ({ children }) => {
  return(
    <Text style = {styles.text}>{children}</Text>
  );
};

const styles = StyleSheet.create({
  text: {
    fontSize: 17,
    color: "#ffffff",
    fontWeight: "600",
  },
});

export default HeaderText;