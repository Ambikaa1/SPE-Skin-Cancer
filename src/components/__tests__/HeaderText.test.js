import React from 'react';
import renderer from 'react-test-renderer';

import HeaderText from "../HeaderText";

describe('<HeaderText />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<HeaderText />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
