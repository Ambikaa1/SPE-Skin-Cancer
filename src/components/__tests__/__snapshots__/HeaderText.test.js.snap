// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HeaderText /> renders correctly 1`] = `
<Text
  style={
    Object {
      "color": "#ffffff",
      "fontSize": 17,
      "fontWeight": "600",
    }
  }
/>
`;
